product_dimension_volumetric_weight/tests/test_product_uom.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+from odoo.tests.common import Form
+
+
+class TestProductUom(common.SavepointCase):
+    def setUp(self):
+        super().setUp()
+        self.uom_m = self.env.ref("uom.product_uom_meter")
+        self.uom_m.write({"weight_volumetric_ratio": 400})
+        self.uom_cm = self.env.ref("uom.product_uom_cm")
+        self.uom_cm.write({"weight_volumetric_ratio": 400})
+
+    def test_volumetric_weight_in_m(self):
+        product_form = Form(self.env["product.template"])
+        product_form.name = "Test product"
+        product_form.product_length = 1
+        product_form.product_height = 1
+        product_form.product_width = 1
+        product_form.dimensional_uom_id = self.uom_m
+        self.assertAlmostEqual(400, product_form.weight_volumetric)
+
+    def test_volumetric_weight_in_cm(self):
+        product_form = Form(self.env["product.template"])
+        product_form.name = "Test product"
+        product_form.product_length = 40
+        product_form.product_height = 50
+        product_form.product_width = 60
```

Comment:
Again testing something that is not of this module. Why don't you set `volume` and just that?